src/crepe/api/Scene.h

```diff
@@ -53,6 +58,55 @@ class Scene {
 	//! Mediator reference
 	OptionalRef<Mediator> mediator;
 	//! \}
+
+protected:
+	/**
+	* \brief Retrieve the reference to the SaveManager instance
+	*
+	* \returns A reference to the SaveManager instance held by the Mediator.
+	*/
+	SaveManager & get_save_manager() const { return mediator->save_manager; }
+
+	/**
+	 * \brief Create a new game object using the component manager
+	 *
+	 * \param name Metadata::name (required)
+	 * \param tag Metadata::tag (optional, empty by default)
+	 * \param position Transform::position (optional, origin by default)
+	 * \param rotation Transform::rotation (optional, 0 by default)
+	 * \param scale Transform::scale (optional, 1 by default)
+	 *
+	 * \returns GameObject interface
+	 *
+	 * \note This method automatically assigns a new entity ID
+	 */
+	GameObject new_object(const std::string & name, const std::string & tag = "",
```

Comment:
Voor alle functies die direct fowarden zou ik gewoon een `\copydoc` gebruiken als documentatie (in dit geval dus `\copydoc ComponentManager::new_object`)

Reply:
is dit nu goed?

Reply:
Ik zou het alleen nog op één regel zetten met `//!` maar voor de rest is het goed